### PR TITLE
C sharp oop 3

### PR DESCRIPTION
**Pull request by KrisAramis**
please review my homework

### CODE REVIEW

File: CSharpOOP2/Car.cs [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+public class Car
+{
+    public string Model;
+    public int ProductionYear = 2020;
+    public bool Diesel;
+    public static bool IsOpen;
+    public string Color;
+
+
```
Comment (dnnyhlwdplc) [inline CSharpOOP2/Car.cs]: Task 1 is not completed :(
I am still able to read and write to all properties from current class

File: Program.cs [hunk 2]
```diff
@@ -1,16 +1,66 @@
...
+
+        #region call method when field is null
+        Car matiz = new Car();
+        matiz.Refuel(40); // no error, by default Filllevel was 0;
+        Console.WriteLine(matiz.FillLevel); // no error, by default Model was null
+        Console.WriteLine(matiz.ReturnModel());
+        #endregion
+        
+        #region Return field with predetermined value without declaring it
+        Car Chevrolet = new Car();
```
Comment (dnnyhlwdplc) [inline Program.cs]: Chevrolet -> chevrolet

File: CShartOOP/Person.cs [hunk 3]
```diff
@@ -0,0 +1,6 @@
+﻿namespace Kristina_Kulich__Application.CShartOOP;
+
+public class Person
```
Comment (dnnyhlwdplc) [inline CShartOOP/Person.cs]: please remove unused classes from PR

File: Program.cs [hunk 4]
```diff
@@ -1,16 +1,66 @@
...
+        //Mazda.VinNumber; Errorline: The property "VinNumber" cannot be used in this context because the get accessor is inaccessible
+        bool Carstate = mazda.EngineIsRunning;
+        #endregion
+
+        #region StaticProperty
+        Car.Beep();
+        Car.IsOpen = true;
+        #endregion
+
+        #region StaticMethod
```
Comment (dnnyhlwdplc) [inline Program.cs:65]: I am unable to see Helper class- :(

**Review by dnnyhlwdplc — CHANGES_REQUESTED** [review]

File: CSharpOOP2/Car.cs [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+        if(FillLevel > 60)
+        {
+            FillLevel = 60;
+        }
+    }
+    public static void Beep()
+     {
+         Console.WriteLine("Beeeeeeep");
+     }
+    public static void RepaintCar(Car car, string newColor = "blue")
```
Comment (dnnyhlwdplc) [inline CSharpOOP2/Car.cs]: this method could be better. We can repaint current object
public static void RepaintCar(string newColor = "blue")
        {
            Color = newColor;
        }

**Review by dnnyhlwdplc — CHANGES_REQUESTED** [review]

File: Program.cs (hunk 4 above)
Comment (KrisAramis) [reply Program.cs:65]: fixed

**Review by KrisAramis — COMMENTED** [review]
please re-review my fixes

File: CSharpOOP2/Car.cs (hunk 5 above)
Comment (KrisAramis) [reply CSharpOOP2/Car.cs]: fixed

File: CShartOOP/Person.cs (hunk 3 above)
Comment (KrisAramis) [reply CShartOOP/Person.cs]: fixed

File: Program.cs (hunk 2 above)
Comment (KrisAramis) [reply Program.cs]: fixed

File: CSharpOOP2/Car.cs (hunk 1 above)
Comment (KrisAramis) [reply CSharpOOP2/Car.cs]: fixed

**Review by dnnyhlwdplc — APPROVED** [review]

**Pull request merged**